components/pthread/Kconfig

```diff
@@ -19,4 +19,21 @@ config PTHREAD_STACK_MIN
     help
         Minimum allowed pthread stack size set in attributes passed to pthread_create
 
+config ESP32_PTHREAD_TASK_CORE_DEFAULT
+    int "Default pthread core affinity"
+    default -1
+    range -1 1
```

Comment:
Has to be conditional on FREERTOS_UNICORE, don't allow setting 1 in single core mode.

Reply:
I'll do something even better - menu selection when unicore is active.